scripts/EdgePy.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+import argparse
+
+from src.DGEList import DGEList
+
+from src.data_import.data_import import DataImporter
+from src.data_import.data_import import GroupImporter
+from src.data_import.data_import import create_DGEList
+from src.data_import.mongodb.mongo_import import ImportFromMongodb
+import configparser
+
+
+def parse_arguments(parser=None):
+    if not parser:
+        parser = argparse.ArgumentParser()
+
+    parser.add_argument("--count_file", help="name of the count file")
+    parser.add_argument("--groups_file", help="name of the groups file")
+    parser.add_argument("--dge_file", help='import from .dge file;')
+    parser.add_argument("--gene_list", default=None, help='a list of genes to filter the data set')
+
+    # mongo parameters
+    parser.add_argument("--mongo_config", help='a way to import data from a supported mongo database')
+    parser.add_argument("--mongo_key_name", default="Project")
+    parser.add_argument("--mongo_key_value", default="RNA-Seq1")
+
+    args = parser.parse_args()
+
+    return args
+
+
+class EdgePy(object):
```

Comment:
This class needs documentation in the form of docstrings. What is it's purpose over a `DGEList`?

Reply:
Add an example in the docstring for a normal invocation and follow [`doctest`](https://docs.python.org/3/library/doctest.html) formatting.

Reply:
DGEList is the class for storing/manipulating data.

This is the command line invocation of edgepy

I will add docstrings.